Dockerfile

```diff
@@ -2,18 +2,18 @@
 
 # Create a first stage container to build the application, this container image will be dropped once
 # the runner is built
-FROM trion/ng-cli:12.2.1 as deps
+FROM trion/ng-cli:17.1.2 AS deps
 
 # This Dockerfile uses labels from the label-schema namespace from http://label-schema.org/rc1/
 LABEL maintainer="clara.schuster@tuwien.ac.at" \
-        org.label-schema.name="DAMAP-frontend" \
-        org.label-schema.description="DAMAP is a tool that aims to facilitate the creation of data management plans (DMPs) for researchers." \
-        org.label-schema.usage="https://github.com/tuwien-csd/damap-frontend/tree/master/README.md" \
-        org.label-schema.vendor="Technische Universität Wien" \
-        org.label-schema.url="https://github.com/tuwien-csd/damap-frontend" \
-        org.label-schema.vcs-url="https://github.com/tuwien-csd/damap-frontend" \
-        org.label-schema.schema-version="1.0" \
-        org.label-schema.docker.cmd="docker run -d -p 8080:8080 damap"
+    org.label-schema.name="DAMAP-frontend" \
+    org.label-schema.description="DAMAP is a tool that aims to facilitate the creation of data management plans (DMPs) for researchers." \
+    org.label-schema.usage="https://github.com/tuwien-csd/damap-frontend/tree/master/README.md" \
+    org.label-schema.vendor="Technische Universität Wien" \
+    org.label-schema.url="https://github.com/tuwien-csd/damap-frontend" \
+    org.label-schema.vcs-url="https://github.com/tuwien-csd/damap-frontend" \
+    org.label-schema.schema-version="1.0" \
+    org.label-schema.docker.cmd="docker run -d -p 8080:8080 damap"
```

Comment:
minor: we can update these with TU Graz specific values
Feel free to just create an issue to tackle later